package.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "@cljs-oss/module-deps": "^1.1.1",
```

Comment:
cljs needs this to process deps?

Reply:
Yes, it was installed when running with `:install-deps true`.